test_sets/PTU_no_certificate_exists_at_module_config.txt

```diff
@@ -0,0 +1,57 @@
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/001_ATF_P_TC_new_navi_app_no_certificate.lua Check_that_SDL_will_trigger_PTU_new_app_registered_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/002_ATF_P_TC_existing_navi_app_no_certificate.lua Check_that_SDL_will_trigger_PTU_existing_app_registered_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/003_ATF_N_TC_new_navi_app_certificate_exist.lua Check_that_SDL_will_trigger_PTU_new_app_registered_certificate_exist
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/004_ATF_N_TC_existing_navi_app_certificate_exist.lua Check_that_SDL_will_not_trigger_PTU_existing_app_registered_certificate_exist
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/005_ATF_P_TC_default_app_first_StartService_RPC_encrypt_true.lua Check_that_SDL_will_triggeer_PTU_when_default_app_sends_first_encrypted_RPC_service
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/006_ATF_P_TC_comm_app_first_StartService_RPC_encrypt_true.lua Check_that_SDL_will_triggeer_PTU_when_comm_app_sends_first_encrypted_RPC_service
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/007_ATF_P_TC_media_app_first_StartService_RPC_encrypt_true.lua Check_that_SDL_will_triggeer_PTU_when_media_app_sends_first_encrypted_RPC_service
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/008_ATF_P_TC_navi_app_PTU_fail_ForceProtectedService_ON_StartService_encrypted_true.lua Check_that_SDL_will_return_StartService_Hybrid_Audio_Video_RPC_NACK_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/009_ATF_P_TC_navi_app_PTU_retry_sequence_finish_ForceProtectedService_ON_StartService_encrypted_true.lua Check_that_SDL_will_return_StartService_Audio_Video_Hybrid_RPC_NACK_when_PTU_fial_because_retry_sequence_finished
```

Comment:
@istoimenova, Scripts 008 - 015 are for req-t, where "ForceProtectedService" parameter is ON
And scripts 027 - 040 are for req-t, where "ForceProtectedService" parameter is OFF
In 1st group there are 8 scripts and in 2nd one - 14 scripts
Could you clarify please why we have different number of scripts in these groups?